### DISCUSSION TITLE
Why is UCI_Elo's min 1350?

### DISCUSSION THREAD

**Discussion by jamesbraza** [official-stockfish/Stockfish#4434] [Q&A] [answered]
From https://en.wikipedia.org/wiki/Chess_rating_system#Elo_rating_system, there are Elo ranges below 1350.

Why does Stockfish have a lower bound on `UCI_Elo` of 1350?

---

Same question here: https://github.com/niklasf/python-chess/issues/746

**Comment by DrGrekenstein** [comment]
https://www.chess.com/forum/view/general/chess-is-random-for-most-people

The Elo of a computer program that plays random moves is 1200, so 1350 is expected.

**Reply by Mercutio1243** [reply]
That's nonsense, no?

**Comment by dav1312** [comment, marked as answer]
I don't think there's really a reason per se. It's just the elo value that is achieved by setting the Skill Level to 0, which is the lowest currently supported by Stockfish. Other Stockfish-based engines, such as Fairy-Stockfish, have modified this value to achieve lower playing strengths.

**Comment by bftjoe** [comment]
You can simulate lower skill levels by decreasing threads, hash, and time.
